tools/rpki-rov.c

```diff
@@ -19,8 +19,10 @@
 const int connection_timeout = 20;
 enum rtr_mgr_status connection_status = -1;
 
-static void connection_status_callback(const struct rtr_mgr_group *group, enum rtr_mgr_status status,
-				       const struct rtr_socket *socket, void *data)
+static void connection_status_callback(const struct rtr_mgr_group *group __attribute__((unused)),
+                                       enum rtr_mgr_status status,
```

Comment:
checkpatch is still not happy with the indention here, running `clang-format -i tools/rpki-rov.c` will fix this.

Reply:
sorry, could you be more specific about the issue I should do:
calling clang-format resulted with some errors
https://gist.github.com/pguibert6WIND/c57c562250f74630e6d79741dc985f61

indeed, I use clang-format 3.8, and I read that I had to move to clang-format 4.0. this is what I said. otherwise I would have had that:
```
YAML:32:21: error: unknown key 'IncludeIsMainRegex'
IncludeIsMainRegex: '(private)?$'
                    ^~~~~~~~~~~~~
Error reading /root/ENV/rtrlib/.clang-format: Invalid argument
```

Reply:
lines 23 and 24 are indented with spaces only, where kernel coding style says you should tabs where possible and spaces when necessary.
Take a look at line 25 for instance.

Reply:
done. thanks